packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -135,35 +123,37 @@ class SectionHeader extends React.Component {
       customProps.className,
     );
 
-    if (!level) {
-      // eslint-disable-next-line no-console
-      console.warn('Default heading level may not appropriate has it would fail to convey context of heading in a site / application where it is used. Heading level should be set explicitly depending on the position of header in site / application to allow screen readers to identify headers consistently.'); // to be removed on next major version release.
-    }
-
-    if (title) {
-      // eslint-disable-next-line no-console
-      console.warn('`title` prop has been renamed to `text`. please update all the refernces of `title` prop to use prop `text`.'); // to be removed on next major version release.
-    }
     const Element = `h${level || 2}`;
```

Comment:
```suggestion
    const Element = `h${level}`;
```
We need to remove default value, As we need to make `level` as required prop, `level` should be specified explicitly to allow screen readers to identify headers consistently.

Reply:
@supreethmr - I don't understand why the prop needs to now be required.  The level will have a value a read the same way in the screen reader.  If the consumer does not want the default value of 2, they can pass the desired value.

Reply:
Could we move the default value to the defaultProps constant? That way it will show up in the documentation as well

Reply:
@cm9361  The reason why we need to make the `level` prop as required is because As part of A11y audit UX team had reported that having default heading level will create inconsistency in heading order. Header elements should always be used with appropriate heading level depending on it's position in a page. Chances application using header components with incorrect heading level is High when we have default value for `level` prop. Hence we had a conclusion to remove default value and make the prop as required. So that consumer will have to provide appropriate heading level to use terra-header components.

This is one of the approach which we have taken for all the terra header components which had heading level. To make this transition of using explicit heading level we had removed value from `defaultProps` and added up it here for passivity concerns.

cc: @scottwilmarth